static/app/utils/useColorscheme.tsx

```diff
@@ -21,7 +21,7 @@ function setFaviconTheme(theme: 'dark' | 'light'): void {
   }
 
   const path = faviconNode.href.split('/sentry/')[0];
-  const iconName = theme === 'dark' ? 'favicon-dark' : 'favicon';
+  const iconName = theme === 'dark' ? 'favicon-dev' : 'favicon';
```

Comment:
I'm not a fan of this. Now there's no easy way to differentiate between production sentry in dark mode and dev mode sentry.

Reply:
Oh that's a good point... ok, I'll get another color